tests/AuthTest.php

```diff
@@ -10,55 +9,56 @@ public function tearDown(): void
     {
         parent::setUp();
         User::truncate();
-        DB::collection('password_reminders')->truncate();
+        DB::collection('password_reset_tokens')->truncate();
     }
 
     public function testAuthAttempt()
     {
         User::create([
             'name' => 'John Doe',
-            'email' => 'john@doe.com',
+            'email' => 'john.doe@example.com',
```

Comment:
Is there any reason why this was test changed?

Reply:
It is not a required change, but just something I made a habit of: `@example.com` was explicitly reserved as a domain for testing purposes. This helps guard against accidental misconfiguration in the test environment sending emails to live domains.

Aside: this is a big problem in Germany, where lots of developers use `@test.de` as test email addresses, not realising that they are actually using the domain of "[Stiftung Warentest](https://en.wikipedia.org/wiki/Stiftung_Warentest)" to test their emails. This has led to some companies getting cease and desist notices from Stiftung Warentest because of abuse of their email system.